packages/entity-detail/src/components/DetailForm/DetailForm.js

```diff
@@ -2,10 +2,14 @@ import React from 'react'
 import {reduxForm} from 'redux-form'
 import {intlShape, FormattedRelative, FormattedMessage} from 'react-intl'
 import {Button, LayoutBox} from 'tocco-ui'
+
+import formErrorsUtil from 'tocco-util/src/form/formErrors'
+import initFormBuilder from 'tocco-util/src/form/formBuilder'
+import {getFieldId} from 'tocco-util/src/form/helpers'
```

Comment:
we usually import tocco-util stuffe like this

import {form} from 'tocco-util'

then

form.formErrors ....